### DISCUSSION TITLE
Show Processing Rate

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1529] [Ideas]
I was thinking of a processing rate in bbot.  For example a number that is average of different processing in seconds. Because sometimes I look at my scan and it seems slow to me, but honestly I don't have any reference to know if it's really slow or not, maybe if there is a number that shows the processing rate it can help us to know if a scan is slow.

**Comment by TheTechromancer** [comment]
Added in https://github.com/blacklanternsecurity/bbot/pull/1530.
